src/components/admin/statistics/GraphAccordionSection.tsx

```diff
@@ -78,17 +78,17 @@ const GraphAccordionSection: React.FC<IProps> = props => (
             Object.keys(genders).length ? (
               <HStack>
                 {Object.keys(universities).length && (
-                  <BarGraphView graphQuestion={graphQuestions.University} data={universities} />
-                )}
-                {Object.keys(ages).length && (
-                  <BarGraphView graphQuestion={graphQuestions.Age} data={ages} />
-                )}
-                {Object.keys(majors).length && (
-                  <BarGraphView graphQuestion={graphQuestions.Major} data={majors} />
-                )}
-                {Object.keys(genders).length && (
-                  <BarGraphView graphQuestion={graphQuestions.Gender} data={genders} />
+                  <TableView heading="Universities" data={universities} />
                 )}
+                <VStack>
+                  {Object.keys(majors).length && (
+                    <TableView heading="Majors" data={majors} />
+                  )}
+                  {Object.keys(schoolYear).length && <BarGraphView heading="School Year" data={schoolYear} />}
+                  {Object.keys(genders).length && (
+                    <PieGraphView heading='Gender' data={genders} />
+                  )}
+                </VStack>
```

Comment:
For these tables, can you sort the data so that it goes most at the top to least at the bottom? It might be easier to do this on the backend and then just send the sorted data to the frontend.